tsconfig-strict.json

```diff
@@ -22,6 +22,7 @@
       "components/*": ["core/templates/components/*"],
       "domain/*": ["core/templates/domain/*"],
       "expressions/*": ["core/templates/expressions/*"],
+      "oppia-events/*": ["core/templates/oppia-events/*"],
```

Comment:
Hmm, should we maybe use just 'events'? We don't prepend oppia to folder names anywhere else, right?

Reply:
'events' has a clash with node default module by the same name. But I get your point. We can still call it events but we would have to use relative path (like starting from `.`) or absolute path (starting from the root of our dir, like core/templates/events). Which one do you think would be better?

Reply:
Ah I see, hmm I'm pretty indifferent but maybe lean towards either having a bit different name for 'events' (maybe 'event') or even keeping the 'oppia' prefix.